java/code/src/com/suse/manager/webui/Router.java

```diff
@@ -75,6 +76,7 @@
     /**
      * Invoked from the SparkFilter. Add routes here.
      */
+    @SuppressWarnings("checkstyle:methodlength")
```

Comment:
The idea here is to suppress checkstyle for the method length because it is intended that this method will grow in line number as soon as we add routes to it